PowerEditor/src/WinControls/ToolBar/ToolBar.cpp

```diff
@@ -408,17 +415,46 @@ void ToolBar::addToRebar(ReBar * rebar)
 	_rbBand.fMask   = RBBIM_CHILD | RBBIM_CHILDSIZE | RBBIM_IDEALSIZE | RBBIM_SIZE;
 }
 
+LRESULT CALLBACK RebarSubclass(
+	HWND hWnd,
+	UINT uMsg,
+	WPARAM wParam,
+	LPARAM lParam,
+	UINT_PTR uIdSubclass,
+	DWORD_PTR dwRefData
+)

```

Comment:
The toolbar of dockable panel & rebar for incremental search are not in dark mode:

![image](https://user-images.githubusercontent.com/90293/110049412-1222e880-7d52-11eb-87fb-082ca3d04cca.png)

Reply:
This one is a bit trickier; the checkboxes are not easily customized unfortunately, but nothing insurmountable

Reply:
Latest commit fixes the find incremental dialog a lot more by added a subclass proc that handles checkboxes. Will need to extend this to edit boxes, though we could use SetWindowTheme with DarkMode_Explorer on it for those on the dark-mode supported Windows 10 builds.  Still need to get the workspace toolbar to be dark but the icons are a bit of a hindrance.

Reply:
Alright, black toolbar for the workspace file view is still workable even if black. I think this is good for now then.

Reply:
Sorry for not revealing that in the last review:

![image](https://user-images.githubusercontent.com/90293/115812072-15049600-a3f1-11eb-928e-87dfc471a09e.png)